Source/WebCore/loader/ContentFilter.h

```diff
@@ -121,6 +126,9 @@ class ContentFilter : public CanMakeWeakPtr<ContentFilter>, public CanMakeChecke
     ResourceError m_blockedError;
     bool m_isLoadingBlockedPage { false };
     bool m_responseReceived { false };
+    unsigned m_platformCompletionHandlersCalled { 0 };
+    unsigned m_platformFiltersAllowed { 0 };
+    std::optional<CompletionHandler<void(ResourceRequest)>> m_willSendRequestCompletionHandler;
```

Comment:
Why is this using `std::optional`? WTF::CompletionHandler already has a null state, isn't it enough, do we need to distinguish std::nullopt vs the CompletionHandler being null somehow?